communication/slack_channels.md

```diff
@@ -0,0 +1,62 @@
+# Slack Channels
+
+Please find a list here of Slack channels at source{d}.
+
+Slack channels are encouraged to be open for everyone to join. Private channels for professional reasons are highly discouraged (have fun with private personal slack channels).
```

Comment:
Worth also mentioning that we should encourage people to have conversations in open/public channels rather than direct messages. People should keep each other in check to push towards transparency. It's not easy but it's important!

Reply:
+1